sources/web/datalab/static/eula.html

```diff
@@ -67,9 +69,11 @@ <h1 class="hero-title text-display-1">Cloud Datalab<sup>Beta</sup></h1>
 
     <div class="body">
       <div class="bodytext">
-        The Google Cloud Platform Standard Terms of Sevice notwithstanding, your use of Google Cloud Datalab Alpha
-        is governed by the <a href="http://cloud-tta.appspot.com/">TTA</a>  and by using Google Cloud Datalab Alpha
-        you agree to its terms.
+        <p>In order to continually innovate and improve our software and identify which additional features are most needed, Google may collect usage statistics when Cloud Datalab is running on your local machine. To turn off the collection of Cloud Datalab usage statistics, click <span class="fa fa-info-circle"></span> at the top of the notebook listing page, select <b>Usage Statistics</b>, uncheck the box and choose <b>Submit</b>.</p>
+        <br />
+        <p>By clicking “Accept” below, you agree that your use of this software is governed by the Google
```

Comment:
That is super odd; thanks for catching that.

I've switched this to just regular quotes.